lib/shared/dictionary.dart

```diff
@@ -3,7 +3,3 @@ const addFavoritesMsg= 'Start adding some favorites ';
 const yourFilter='Your Filters';
 const yourFavorites='Your Favorites';
 const categories ='Categories';
-
```

Comment:
😞 see we only need 1 newline so just **press enter 1 time only**.

Reply:
6 line now. need one more?

Reply:
only 1 after line number 5

Reply:
no 6 lines

Reply:
in my Text Editor is already 5 line and one line blank

Reply:
just have 1 more and then go check in your repo's file. Now there is no extra line at the end. you can recheck that